u2f-ref-code/java/src/com/google/u2f/server/impl/U2FServerReferenceImpl.java

```diff
@@ -158,7 +158,8 @@ public SecurityKeyData processRegistrationResponse(RegistrationResponse registra
 
     Log.info("Verifying signature of bytes " + Hex.encodeHexString(signedBytes));
     if (!cryto.verifySignature(attestationCertificate, signedBytes, signature)) {
-      throw new U2FException("Signature is invalid");
+      //throw new U2FException("Signature is invalid");
```

Comment:
No, didn't mean to change.  Android attestations won't verify for now, so this was part of my test code.  Thanks for catching.